src/components/Popups/PopupReply.tsx

```diff
@@ -36,31 +35,28 @@ export default function PopupReply({
     };
     const handleCancel = (event: React.MouseEvent): void => {
         handleEvent(event);
-        onCancel(text);
+        onCancel(value);
```

Comment:
Do we need to call it with `value`? `handleCancel` and `handleSubmit` in `RegionAnnotations` have 0 arguments.

Reply:
Is there a strong reason not to call it with `value`? It allows the parent to decide what to do with it.